test/core/end2end/tests/max_concurrent_streams.cc

```diff
@@ -133,6 +133,11 @@ CORE_END2END_TEST(Http2SingleHopTest, MaxConcurrentStreams) {
   live_call = (live_call == 300) ? 400 : 300;
   Expect(live_call + 1, true);
   Step();
+  // Spin for a little: we expect to see no events now - and this time allows
+  // any overload protection machinery in the transport to settle (chttp2 tries
+  // to ensure calls are finished deleting before allowing more requests
+  // through).
```

Comment:
IIUC, this is indirectly adding a 1s delay since there are no active expected events. Why not do the sleep explicitly?

Reply:
Doing it this way means that the sleep is controlled by the fuzzing infrastructure too, so ultimately most of the time we scan just skip time forward by a second.